preprocessor/processor.py

```diff
@@ -0,0 +1,56 @@
+from pathlib import Path
+import os
+import glob
+import cv2 as cv
```

Comment:
Please also add a `requirements.txt` that lists the version of cv2 that the script needs.